### PR TITLE
Update mediaBucket values

### PR DESCRIPTION
**Pull request by tomgiddingsjisc**
**Infra changes**

- Added tfvar files to the repository (we never use these to store sensitive values, and committing these would go against best practice anyway).
- Added `int.tfvars` file
- Set iam policy for s3 public access

**API changes**

Added `iamRoleStatement` (at request of @nathan-at-jisc) for lambda and email functions.

**UI changes**

Fixed the s3 bucket url for user uploaded content (eg. publication editor)

Currently object src are along the lines of...
https://science-octopus-publishing-images-int.eu-west-1.amazonaws.com//cl2yizbzd000009mr0dr62uhv

Where they need to be:
https://science-octopus-publishing-images-int.s3.eu-west-1.amazonaws.com/cl2yizbzd000009mr0dr62uhv 

This fix removes the surplus slash, and adds `s3` to the bucket url.

---

### Acceptance Criteria:

n/a

---

### Tests:

No additional

---

### Screenshots:

n/a

### CODE REVIEW

**Review by AshleyRedmanJisc — APPROVED** [review]
The change looks good to me, imagining this doesn't effect local & fixes the 'non-local' url for the bucket(s)? If so, then lgtm.

**Comment by tomgiddingsjisc** [comment]
> The change looks good to me, imagining this doesn't effect local & fixes the 'non-local' url for the bucket(s)? If so, then lgtm.

@AshleyRedmanJisc Thanks - I'd managed to miss off the terraform config so that's now added.

This should only impact the non-local buckets (and wouldn't have been something we'd spot locally!)

**Review by AshleyRedmanJisc — DISMISSED** [review]
Ah, didn't realise there was aws config / it wasn't publicly accessible by default, but this data part of the module looks good to me.

File: infra/create-app/int.tfvars [hunk 1]
```diff
@@ -0,0 +1,8 @@
+domain_name = "int.octopus.ac"
```
Comment (AshleyRedmanJisc) [inline infra/create-app/int.tfvars:1]: If i remember right, we're not suppose to include these values in the repo? Treating them like .env

Comment (AshleyRedmanJisc) [reply infra/create-app/int.tfvars:1]: Ok cool, thanks for clarifying 👍🏼

**Review by AshleyRedmanJisc — DISMISSED** [review]

**Comment by AshleyRedmanJisc** [comment]
@tomgiddingsjisc Is this ready for a re-review or is this still on going?

**Comment by tomgiddingsjisc** [comment]
> @tomgiddingsjisc Is this ready for a re-review or is this still on going?

Sorry, this is good for review again now. Needed some tweaking.

**Review by AshleyRedmanJisc — APPROVED** [review]
iam policies are a little mysterious, but having to create one to allow the bucket to be publicly accessible makes sense. Assuming the changes pass a terraform plan, then i'm happy with the changes.

**Review by nathan-at-jisc — APPROVED** [review]
lgtm

**Pull request merged**